### PR TITLE
I1 drivers accident report

### PR DESCRIPTION
**Pull request by YardenIS388**
1. UI skeleton is standing
2. Wizard was completed
3. photo gallery upload component working
4. add signature completed
5. no document uploader
6. no exit button

### CODE REVIEW

File: src/screens/AccidentReport/components/CameraButton.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  });
+
+  return (
+    <>
+      <Pressable
+        style={[style, styles.cameraButoonContainer]}
+        onPress={() => setModalVisible(true)}>
+        <View>
+          <Image
+            source={require('../../../assets/accidentReport/icon3.png')}
```
Comment (henaharon) [inline src/screens/AccidentReport/components/CameraButton.js:109]: icons should be in uiKit/icons

File: src/screens/AccidentReport/components/CameraButton.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            style={styles.cameraIcon}
+          />
+        </View>
+      </Pressable>
+      {/* <Text>{JSON.stringify(response)}</Text> */}
```
Comment (henaharon) [inline src/screens/AccidentReport/components/CameraButton.js:114]: remove unused code

File: src/screens/AccidentReport/components/HelpModal.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        <Text style={[styles.helpBtn, styles.whiteText]}> עזרה</Text>
+        <Image source={accidentReport.help} style={styles.helpIcon}></Image>
+      </Pressable>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  centeredView: {
+    // display: 'flex',
```
Comment (henaharon) [inline src/screens/AccidentReport/components/HelpModal.js:73]: remove unused code

File: src/screens/AccidentReport/components/HelpModal.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    justifyContent: 'center',
+    alignItems: 'center',
+    // marginTop: 22,
```
Comment (henaharon) [inline src/screens/AccidentReport/components/HelpModal.js:76]: remove unused code

File: src/screens/AccidentReport/components/PhotosGallery.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import {useState} from 'react';
+import {Image, ScrollView, StyleSheet, View} from 'react-native';
+import SVG from 'react-native-svg';
+// import MySVG from '../../../assets/accidentReport/icon6.svg';
```
Comment (henaharon) [inline src/screens/AccidentReport/components/PhotosGallery.js:4]: remove unused code

File: src/screens/AccidentReport/components/PhotosGallery.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+        {imagesArr.map((currImage, index) => {
+          console.log(currImage + index + 'currImage + index');
+
+          return (
+            <View key={currImage + index} style={styles.imageContainer}>
+              <Image source={{uri: currImage}} style={styles.image} />
+              <View style={styles.iconContainer}>
+                <Image
+                  source={require('../../../assets/accidentReport/icon6.png')}
```
Comment (henaharon) [inline src/screens/AccidentReport/components/PhotosGallery.js:29]: icons should be in uikit/icons

File: src/screens/AccidentReport/components/Signiture.js [hunk 7]
```diff
@@ -0,0 +1,111 @@
+import {useRef} from 'react';
+import {StyleSheet, View, Text, Pressable} from 'react-native';
+import SignatureScreen from 'react-native-signature-canvas'
+
+ const Signiture = ({text, onOK}) => {
```
Comment (henaharon) [inline src/screens/AccidentReport/components/Signiture.js:5]: signature not working on android, video sent over slack

File: src/screens/AccidentReport/components/Wizard.js [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.buttonContainer}>
+      <Pressable style={styles.button} title="Next" onPress={goToNextPage} disabled={currentPage > (children.length -1) && true}>
+            <Image source={loginIcons.loginButtonArrow}></Image>
+      </Pressable>
+      <View>
+          <Text style={{color:'white', textAlign:"center"}}>
```
Comment (henaharon) [inline src/screens/AccidentReport/components/Wizard.js:35]: no inline style

File: src/screens/AccidentReport/components/Wizard.js [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+            {titles[currentPage]}
+          </Text>   
+          <View style={{flexDirection:'row-reverse', gap:5}}>
```
Comment (henaharon) [inline src/screens/AccidentReport/components/Wizard.js:38]: no inline style

File: src/screens/AccidentReport/components/Wizard.js [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+import { accidentReport, loginIcons} from '../../../uiKit/icons';
+
+const WizardSection = ({ children }) => {
+  return <View style={styles.sectionContainer}>{children}</View>;
+};
+
+const FormWizard = ({ children, startPage }) => {
+  const [currentPage, setCurrentPage] = useState(startPage ? startPage : 0);
+
+  const goToPreviousPage = () => {
```
Comment (henaharon) [inline src/screens/AccidentReport/components/Wizard.js:12]: left arrow should be pink, capture sent over slack

**Comment by YardenIS388** [comment]
Hi, I did my best to solve all of the review items, the signature is still not working on android. Really sorry but it's the best I can do now. 
I hope when I finish all my tests i can go over it again and make it perfect (for us not for a grade)

Thank you so much for the course. It was a blast !

**Pull request merged**